### PR TITLE
Add buttons to 'Cleanup caches and update paths' in standard way

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
Change to standard method to add buttons to this form

Before
----------------------------------------
Added using hardcoded method on form.  Not using standard formButtons template.

After
----------------------------------------
Added using standard formButtons method.
image

Technical Details
----------------------------------------

Comments
----------------------------------------
This allows themes such as haystack (shown) to theme buttons consistently.

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@demeritcowboy this might be one you could look at if you get some headspace

File: CRM/Admin/Form/Setting/UpdateConfigBackend.php [hunk 1]
```diff
@@ -42,22 +42,22 @@ class CRM_Admin_Form_Setting_UpdateConfigBackend extends CRM_Admin_Form_Setting
...
-      ['class' => 'crm-form-submit', 'id' => 'resetpaths']
-    );
-
-    //parent::buildQuickForm();
+    $this->addButtons([
+      [
+        'type' => 'next',
+        'name' => ts('Cleanup Caches'),
+        'subName' => 'cleanup',
+      ],
```
Comment (demeritcowboy) [inline CRM/Admin/Form/Setting/UpdateConfigBackend.php]: Not sure if it was intentional to remove the custom icons? If not then just need to add `'icon' => 'fa-undo',` to the array, and `'icon' => 'fa-terminal',` below.

**Comment by demeritcowboy** [comment]
* General standards
    * [`r-explain`] __PASS__
    * [`r-user`] __PASS__
    * [`r-doc`] __PASS__
    * [`r-run`] __PASS__ Works.
* Developer standards
    * [`r-tech`] __PASS__
    * [`r-code`] __Issue__ Just the icons mentioned above, unless it was intentional.
    * [`r-maint`] __N/A__
    * [`r-test`] __PASS__

**Comment by eileenmcnaughton** [comment]
@mattwire I'm happy for you to merge this once you've considered @demeritcowboy's feedback as it appears to be 'optional' feedback with an overall 'pass' (or even B+ :))

File: CRM/Admin/Form/Setting/UpdateConfigBackend.php (hunk 1 above)
Comment (mattwire) [reply CRM/Admin/Form/Setting/UpdateConfigBackend.php]: Not it was not intentional - I missed them.  Thankyou.

**Pull request merged**